File: Difficulty: Easy/String Duplicates Removal/string-duplicates-removal.java
// User function Template for Java

class Solution {
    String removeDuplicates(String s) {
       boolean[] freq = new boolean[256];
       StringBuilder result = new StringBuilder();

       for(int i=0;i<s.length();i++){
           char ch=s.charAt(i);
           int idx=(int) ch;
           if(!freq[idx]){
               result.append(ch);
               freq[idx]=true;
           }
       }
        return result.toString();
    }
}

File: Difficulty: Easy/String Duplicates Removal/string-duplicates-removal.cpp
// User function template for C++
class Solution {
  public:
    string removeDuplicates(string &s) {
        // code here
        unordered_set<char> freq;
        string result=" ";
        for(char c:s){
            if(freq.find(c)==freq.end()){
                result.push_back(c);
                freq.insert(c);
            }
        }
        return result;
    }
};